src/Codeception/Coverage/SuiteSubscriber.php

```diff
@@ -61,6 +61,17 @@ protected function applySettings($settings)
                 $this->settings[$key] = $settings['coverage'][$key];
             }
         }
+
+        if (isset($settings['coverage']['advanced'])) {
+            foreach ($settings['coverage']['advanced'] as $key => $value) {
+                $method = 'set' . ucfirst($key);
```

Comment:
All `set` methods will be removed in php-code-coverage 9.0.0 - https://github.com/sebastianbergmann/php-code-coverage/blob/master/ChangeLog.md#removed

Reply:
Ah, just checked the changes, will be simple to accommodate them.

Reply:
While the changes are simple to accomodate it means we must require php 7.3 and phpunit 9.3.
I am fine with upping this requirement, but I know others are not (@Naktibalda).

I will not spend time on creating multiple code paths just to accomodate older software versions as it goes against most of what I believe in; so I'm closing this PR.

Reply:
What about upping the version to justify upping the requirements? We're using Codeception in our projects whith parallel tests by splitting tests in group. That's crazy fast and convenient. And it's impossible to use phpCodeCoverage as it is now in the stable version of Codeception because we're getting incorrect results when merging code coverage. So we're stucked with using our fork with hacks.
Personally I don't see any problems requiring latest version of the main testing package and especially version of the language that was published almost 2 years ago.
Please consider adding this options of codeCoverage package as it is severly needed by many developers.

Reply:
I agree; but the reality is that the main contributors on Codeception need to support older PHP versions. 
We do not have the capacity to maintain multiple versions at this time.